### ISSUE TITLE
`soffice` command results in an error if the filename contains special characters.

### ISSUE THREAD

**Issue by jonathandixon** [samvera/hydra-derivatives#163]
When running `Hydra::Derivatives::DocumentDerivatives` with a file name that includes parenthesis `soffice` exits with an error.

```
Unable to execute command "soffice --invisible --headless --convert-to pdf --outdir /tmp /tmp/file(1).pptx".
Exit code: pid 18230 exit 2 Error message: sh: 1: Syntax error: "(" unexpected
```

**Comment by jonathandixon** [comment]
I wonder if this is as simple as quoting `#{path}` on this [line](https://github.com/samvera/hydra-derivatives/blob/ade91910414a1821c110d97a441941be0c1ee6ca/lib/hydra/derivatives/processors/document.rb#L6)

```
execute "#{Hydra::Derivatives.libreoffice_path} --invisible --headless --convert-to #{format} --outdir #{outdir} \"#{path}\""
```

I'm not sure it this will have unwanted consequences. Should there also be a check to determine if the path is already quoted?

**Comment by mjgiarlo** [comment]
@jonathandixon I wonder if we should run the path through a method like https://ruby-doc.org/stdlib-2.3.0/libdoc/shellwords/rdoc/Shellwords.html#method-c-shellescape

**Comment by cjcolvar** [comment]
Fixed by #164.
